docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -47,27 +47,33 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
     - **Role**: Enter your Snowflake role.
 
-    - **Authenticator**: Enter the Snowflake database authenticator that you want to use to verify your Snowflake connection. 
+6. Optional. Make the following changes:
 
-5. Optional. Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
+    - Clear **Create temp table** if you don't want to create a temporary database table. Temporary database tables store data temporarily and can improve performance by making queries run faster.
 
-6. Optional. Clear **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
+    - Select **Test connection** if you don't want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```

Comment:
```suggestion
    - Select **Test connection** if you want to test the Data Asset connection. Testing the connection to the Data Asset is a preventative measure that makes sure the connection configuration is correct. This verification can help you avoid errors and can reduce troubleshooting downtime.
```
Shouldn't they be selecting **Test connection** if they do want to test the Data Asset connection, or leave it unselected it if they do not?